### PR TITLE
chore: dynamically add the year using JavaScript

### LINKED ISSUE #23: Inside the footer dynamically add the year using JavaScript
For the footer add this span inside the p element 
```
    <footer>
      <div class="container">
        <p>Copyright &copy; <span id="copyright"></span></p>
      </div>
    </footer>
```
Then in the JavaScript file, target that id, and use the Date function to display the full year. 

This should be the result
Screen Shot 2022-08-07 at 4 39 44 PM

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getFullYear

### PR DESCRIPTION
**Pull request by DBenMoshe**
Added dynamic copyright date using Javascript to target the HTML

closes #23

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -58,5 +58,7 @@ const giveRatio = () => {
   }
 };
 
+document.querySelector("#copyright").innerText = new Date().getFullYear();
```
Comment (AndreyDmitriev39r) [inline index.js:61]: This looks good, tho I have little experience with DOM. Whole thing looks good in the browser also.
Question: maybe we should place these line related to footer below all stuff related to contrast ratio, not in the middle of it?

**Review by AndreyDmitriev39r — CHANGES_REQUESTED** [review]

**Comment by AndreyDmitriev39r** [comment]
@kernix I guess you have more experience with DOM than me, requested additional review from you.

File: index.html [hunk 2]
```diff
@@ -123,7 +123,7 @@ <h4>AAA level</h4>
 
     <footer>
       <div class="container">
-        <p>&copy;</p>
+        <p>Copyright &copy; <span id="copyright"></span></p>
         <!--TODO: Use JavaScript to dynamically add the current year here-->
```
Comment (jdwilkin4) [inline index.html:127]: You can remove this comment since the change has been implemented

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (jdwilkin4) [reply index.js:61]: It doesn't hurt to place it there, but organizationally it would be nice to place it at the bottom

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Move your code at line 61 down below the contrast ratio functions

**Review by AndreyDmitriev39r — APPROVED** [review]
LGTM!

**Pull request merged**